tests/test_pdf_chisquared.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright (c) 2024 zfit
+import numpy as np
+import pytest
+
+import zfit
+from zfit import Parameter
+from zfit.models.dist_tfp import ChiSquared
+
+N_true = 2
+
+obs = zfit.Space(obs="Nobs", limits=(0, 200))
```

Comment:
create the obs and test values inside the test function (and use znp.name instead of np.name wherever possible)

Reply:
Ok sure.
For my curiosity: they are global because I started from a copy of [test_pdf_poisson.pdf](https://github.com/zfit/zfit/blob/develop/tests/test_pdf_poisson.py). Is this intended there because there is more than one function accessing the values? Or is it just legacy and no-one's gotten around to change it?